lib/puppet/provider/registry_key/registry.rb

```diff
@@ -37,13 +37,7 @@ def destroy
     Puppet.debug("Destroying registry key #{self}")
```

Comment:
When using `puppet resource` I'm getting an error like:


```
C:\source\puppetlabs-registry [pr/146 +7 ~0 -0 !]> bundle exec puppet resource registry_key 'HKLM\Software\Vendor\PuppetLabsTest_Value_Glenn2\SubKey1' --modulepath .\spec\fixtures\modules\
Error: Could not run: undefined local variable or method `hkeys' for Puppet::Type::Registry_key::ProviderRegistry:Class
C:\source\puppetlabs-registry [pr/146 +7 ~0 -0 !]> git checkout master
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
C:\source\puppetlabs-registry [master...origin/master +7 ~0 -0 !]> bundle exec puppet resource registry_key 'HKLM\Software\Vendor\PuppetLabsTest_Value_Glenn2\SubKey1' --modulepath .\spec\fixtures\modules\
registry_key { 'HKLM\Software\Vendor\PuppetLabsTest_Value_Glenn2\SubKey1':
  ensure => 'present',
}
```

Reply:
Fixed in the "refactor Provider Base as a Module" commit